### ISSUE TITLE
Add support for exporting Breaker to CSV, XLS etc

### ISSUE THREAD

**Issue by phochste** [LibreCat/Catmandu-Breaker#2]
Like in:

```
$ catmandu breaker data.breaker to XLS --opt1 val1 --opt2 val2 > stats.xls
```

**Comment by nichtich** [comment]
With option `--as` like [Catmandu::Exporter::Stat](https://metacpan.org/pod/Catmandu::Exporter::Stat)?

First step would be to change the internal format

    <record-identifier><tab><metadata-field><tab><metadata-value><tab><metadata-value>...

to a record format

~~~
identifier: ...
field: ...
value: ...
~~~

and use header-less TSV exporter as default. I don't quite understand why "metadata-value" is repeatable, is there a difference between

    <record-identifier><tab><metadata-field><tab><metadata-value><tab><another-metadata-value>...

and

    <record-identifier><tab><metadata-field><tab><metadata-value>
    <same-record-identifier><tab><same-metadata-field><tab><another-metadata-value>

?

**Comment by phochste** [comment]
Available in 0.14
